modules/sdk/gradle-plugins-workspace/src/main/resources/schemas/client-extension.schema.json

```diff
@@ -217,6 +217,33 @@
 					"allOf": [
 						{
 							"$ref": "#/definitions/propertySets/properties/url"
+						},
+						{
+							"properties": {
+								"scriptElementAttributes": {
+									"additionalProperties": {
+										"type": [
+											"boolean",
+											"integer",
+											"number",
+											"string"
+										]
+									},
+									"description": "A list of HTML attributes that will be added to the script element.",
+									"not": {
+										"allOf": [
+											{
+												"required": [
+													"src"
+												]
+											}
+										]
+									},
+									"title": "Script Element Attributes",
+									"type": "object"
```

Comment:
There is a whole discussion about this part of the schema. Refer to https://github.com/liferay-platform-experience/liferay-portal/pull/492/commits/d52d7e41743ccb4359c07f589f5e44b87a2eeff2#r1509646275